README.md

```diff
@@ -31,7 +31,7 @@ go get github.com/intelligentpos/structextract
 		Field4: 123,
```

Comment:
GLOBAL (i.e. all files): The return type is referred to in README and comments as an array, but seems like it's always a slice. Consider correcting? 
README (i.e. this file) only: Is indentation in code snippets 'out'?